lib/model/model.go

```diff
@@ -1780,6 +1768,10 @@ func (m *Model) internalScanFolderSubdirs(ctx context.Context, folder string, su
 		return err
 	}
 
+	defer func() {
+		runner.SchedulePull()
+	}()
```

Comment:
`defer runner.SchedulePull()`

Reply:
Also, what's the point of pulling after a scan?
Just to do the sequence checks?

Reply:
E.g. for the scenario where puller is failing due to a file1 (whatever reason, e.g. permission error). User notices this in logs, resolves it manually and the scan pick the change up. Generally it changes the index so check that there is nothing changed in our needs.